src/components/login.js

```diff
@@ -9,8 +9,8 @@ function Login() {
   return (
     <div>
       <SpotifyAuth
-        redirectUri="https://unpawsthemusic.vercel.app/redirect"
-        clientID={process.env.REACT_APP_API_KEY}
```

Comment:
probably meant to revert this?

Reply:
couldn't check the deployed vercel preview bc it couldn't find localhost

Reply:
whoops